test/controllers/notifications_controller_test.rb

```diff
@@ -542,10 +542,10 @@ class NotificationsControllerTest < ActionDispatch::IntegrationTest
   test 'search results can filter by muted:false' do
     sign_in_as(@user)
     Notification.destroy_all
-    create(:notification, user: @user)
+    create(:notification, user: @user, muted_at: Time.current)
     create(:notification, user: @user)
     get '/?q=muted%3Afalse'
-    assert_equal assigns(:notifications).length, 2
```

Comment:
@andrew `muted:false` will be treated as no muted option provided at all.  So `notifcations.length` will be 2.

Reply:
The behaviour that we’d like is for `muted:false` to not return muted notifications

Reply:
ok. will update the code on both PR's